### PR TITLE
row_event.go: fix the set bytes value in revert order

### PR DESCRIPTION
**Pull request by july2993**
fix https://github.com/pingcap/tidb-tools/issues/70

according to the binlog, we should handle it in little end 
i have test more when the set num is 18 (need 3 bytes), but i can't find a doc from mysql about this

### CODE REVIEW

**Comment by GregoryIan** [comment]
if no document about it, we should test `enum`, `bit` and `set` in  common versions.  one of our intern is working on it

**Comment by siddontang** [comment]
should add a test @july2993

**Comment by Tom-Xie** [comment]
Similar bug occurs when handle `MYSQL_TYPE_ENUM` which is more using more than one bytes to represent enum elements. Thus, it should decode in little-endian form.

**Comment by july2993** [comment]
will fix it later and add test

**Comment by july2993** [comment]
@siddontang PTAL

File: replication/row_event_test.go [hunk 1]
```diff
@@ -523,6 +523,71 @@ func (_ *testDecodeSuite) TestParseJsonDecimal(c *C) {
 	}
 }
 
+func (_ *testDecodeSuite) TestEnum(c *C) {
```
[the remark this answers is not in the document]
Comment (july2993) [reply replication/row_event_test.go]: 😯 i make a mistake, should add more than 256 enum num but not more than 8 to test case length=2

[the remark this answers is not in the document]
Comment (siddontang) [reply replication/row_event_test.go]: please add a new test later @july2993

**Pull request merged**